File: POOUber/Java/car.java
class Car {
    Integer id;
    String license;
    account driver;
    private Integer passeger;

    public Car(String license, account driver){
        this.license = license;
        this.driver = driver;
    }

    void printDataCar() {
        System.out.println("License: " + license + " Name Driver: " + driver.name + "passeger: " + passeger);
    }

}

File: POOUber/JS/car.js
function car(licencia, conductor) {
    this.id;
    this.license = licencia;
    this.driver = conductor;
    this.passenger;
}
car.prototype.printDataCar = function()
{
    console.log(this.driver)
    console.log(this.driver.name)
    console.log(this.driver.document)

}